src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImport.java

```diff
@@ -308,6 +308,40 @@ public void execute(final String formatString)
             }
          });
    }
+
+   private void promptForFactorString(
+      String title,
+      final Operation complete,
+      final String columnName,
+      final String columnType)
+   {
+      globalDisplay_.promptForText(
+         title,
+         "Please a comma separated list of factors",
```

Comment:
Missing `"insert"` or something similar? I also wonder if we could piggyback on Shiny and use `selectize.js` or something here.